shaders/src/light_indirect.fs

```diff
@@ -676,14 +671,26 @@ void applyRefraction(
     float lod = max(0.0, 2.0 * log2(tweakedPerceptualRoughness) + frameUniforms.refractionLodOffset);
 
     vec4 Fat = textureLod(light_ssr, p.xy, lod);
-    float at = Fat.a;
-#if defined(IS_TRANSPARENT_OR_REFRACTIVE)
-    vec3 Ft = mix(vec3(1.0), Fat.rgb, at);
-#else
-    vec3 Ft = Fat.rgb;
 #endif
 
-#endif
+    vec3 Ft;
+    float at;
+    if (frameUniforms.needsAlphaChannel == 1.0) {
+        // Bias transparent view using the following formula:
+        // https://twitter.com/garrettkjohnson/status/1526645080525705216
+        const vec4 backgroundColor = vec4(1.0);
```

Comment:
Why is it all white? What do `color1` and `color2` mean in the context of that formula?